etsi_its_msgs/etsi_its_cpm_ts_msgs/msg/WrappedCpmContainer.msg

```diff
@@ -26,21 +25,37 @@
 # --- Auto-generated by asn1ToRosMsg.py ----------------------------------------
 
 # --- ASN.1 Definition ---------------------------------------------------------
+# /**
+#  * This DF represents a CPM container preceded by its type identifier and a lenght indicator.
+#  *
+#  * It shall include the following components: 
+#  *
+#  * @field containerId: the identifier of the container type.
+#  *
+#  * @field containerData: the container content consistent with the container type.
+#  *
+# */
 # WrappedCpmContainer ::= SEQUENCE {
 #    containerId     CPM-CONTAINER-ID-AND-TYPE.&id( {CpmContainers} ),
+#    containerData   CPM-CONTAINER-ID-AND-TYPE.&Type( {CpmContainers}{@containerId} )
+# }
 # ------------------------------------------------------------------------------
-# *
-# * This DF represents a CPM container preceded by its type identifier and a lenght indicator.
-# *
-# * It shall include the following components: 
-# *
-# * @field containerId: the identifier of the container type.
-# *
-# * @field containerData: the container content consistent with the container type.
-# *
-#
 
 CpmContainerId container_id
 
-CpmContainers container_data
+
+OriginatingVehicleContainer container_data_originating_vehicle_container
+uint8 CHOICE_CONTAINER_DATA_ORIGINATING_VEHICLE_CONTAINER = 0
+
+OriginatingRsuContainer container_data_originating_rsu_container
+uint8 CHOICE_CONTAINER_DATA_ORIGINATING_RSU_CONTAINER = 1
+
+SensorInformationContainer container_data_sensor_information_container
+uint8 CHOICE_CONTAINER_DATA_SENSOR_INFORMATION_CONTAINER = 2
+
+PerceptionRegionContainer container_data_perception_region_container
+uint8 CHOICE_CONTAINER_DATA_PERCEPTION_REGION_CONTAINER = 3
+
+PerceivedObjectContainer container_data_perceived_object_container
+uint8 CHOICE_CONTAINER_DATA_PERCEIVED_OBJECT_CONTAINER = 4
```

Comment:
Choice IDs are not set correctly, leading to erroneous encoding/decoding in conversion.